Sender: LSF System <lsfadmin@eu-g3-014>
Subject: Job 217032066: <synthdata> in cluster <euler> Exited

Job <synthdata> was submitted from host <eu-login-08> by user <jmattern> in cluster <euler> at Tue May  3 13:29:29 2022
Job was executed on host(s) <20*eu-g3-014>, in queue <gpu.24h>, as user <jmattern> in cluster <euler> at Tue May  3 13:29:49 2022
</cluster/home/jmattern> was used as the home directory.
</cluster/work/sachan/jmattern/private-datasets> was used as the working directory.
Started at Tue May  3 13:29:49 2022
Terminated at Tue May  3 13:29:51 2022
Results reported at Tue May  3 13:29:51 2022

Your job looked like:

------------------------------------------------------------
# LSBATCH: User input
python train_classifier.py --gen-data --device 0
------------------------------------------------------------

Exited with exit code 1.

Resource usage summary:

    CPU time :                                   0.22 sec.
    Max Memory :                                 14 MB
    Average Memory :                             -
    Total Requested Memory :                     20480.00 MB
    Delta Memory :                               20466.00 MB
    Max Swap :                                   -
    Max Processes :                              -
    Max Threads :                                -
    Run time :                                   24 sec.
    Turnaround time :                            22 sec.

The output (if any) follows:

  File "train_classifier.py", line 99
    model=model.to(f'cuda:{args.device}')
                                       ^
SyntaxError: invalid syntax
Sender: LSF System <lsfadmin@eu-g3-014>
Subject: Job 217032322: <synthdata> in cluster <euler> Exited

Job <synthdata> was submitted from host <eu-login-08> by user <jmattern> in cluster <euler> at Tue May  3 13:31:12 2022
Job was executed on host(s) <20*eu-g3-014>, in queue <gpu.24h>, as user <jmattern> in cluster <euler> at Tue May  3 13:31:47 2022
</cluster/home/jmattern> was used as the home directory.
</cluster/work/sachan/jmattern/private-datasets> was used as the working directory.
Started at Tue May  3 13:31:47 2022
Terminated at Tue May  3 13:31:52 2022
Results reported at Tue May  3 13:31:52 2022

Your job looked like:

------------------------------------------------------------
# LSBATCH: User input
python train_classifier.py --gen-data --device 0
------------------------------------------------------------

Exited with exit code 127.

Resource usage summary:

    CPU time :                                   0.19 sec.
    Max Memory :                                 13 MB
    Average Memory :                             3.00 MB
    Total Requested Memory :                     20480.00 MB
    Delta Memory :                               20467.00 MB
    Max Swap :                                   -
    Max Processes :                              1
    Max Threads :                                1
    Run time :                                   30 sec.
    Turnaround time :                            40 sec.

The output (if any) follows:

python: error while loading shared libraries: libpython3.8.so.1.0: cannot open shared object file: No such file or directory
